src/AppBundle/Controller/OrderController.php

```diff
@@ -42,11 +41,11 @@ private function createOrderFromRequest(Request $request)
             $order->addCartItem($item, $menuItem);
         }
 
+        // TODO Use OrderManager::applyTaxes
```

Comment:
Why not use it right away?

Reply:
Sorry wrong comment, this is actually useless here.